pkg/hostsfile/hostsfile.go

```diff
@@ -11,10 +11,31 @@ type HostsEntry struct {
 	hosts []string
 }
 
-func (he HostsEntry) String() string {
+func (he *HostsEntry) String() string {
 	return fmt.Sprintf(strings.Join(append([]string{he.ip}, he.hosts...), "\t"))
 }
 
+func (thisp *HostsEntry) Equals(otherp *HostsEntry) bool {
```

Comment:
Can't find a default interface for this. I figured it would already exist as a stdlib interface. This does the trick, but feels a bit ad-hoc.